packages/fetchai/contracts/erc1155/contract.py

```diff
@@ -60,6 +60,15 @@ def create_item_ids(self, token_type: int, token_ids: List[int]) -> None:
         for token_id in token_ids:
```

Comment:
Why do we have these two functions? Should we not just have one?

Reply:
No it is a different functions. The create_item_ids_based_on_nb_goods generates based on the amount of items we want to create the same way. you generate them in  TAC. The other one need the list of item ids.

Reply:
And also let me generate the item_ids if we have an already deployed contract.